griptape/drivers/sql/amazon_redshift_sql_driver.py

```diff
@@ -0,0 +1,106 @@
+import time
+from typing import Optional
+import boto3
+from griptape.drivers import BaseSqlDriver
+from attr import define, field
+
+
+@define
+class AmazonRedshiftSqlDriver(BaseSqlDriver):
+    database: str = field(kw_only=True)
+    cluster_identifier: str = field(default=None, kw_only=True)
+    workgroup_name: str = field(default=None, kw_only=True)
+    db_user: str = field(default=None, kw_only=True)
+    secret_arn: str = field(default=None, kw_only=True)
+    region_name: str = field(default="us-east-1", kw_only=True)
+    wait_sec: float = field(default=0.3, kw_only=True)
+    client = field(default=None, kw_only=True)
```

Comment:
If we use session then we can instantiate a default client from that like we [do it](https://github.com/griptape-ai/griptape-tools/blob/main/griptape/tools/aws_s3_client/tool.py#L12) in tools.

Reply:
Got it, I like that. I'll update this to use `session`. I have the `client` as an optional parameter so that the unit tests can pass in a stubbed client.